### PR TITLE
[Docs] : Update and improve readme files in Resources

### PR DESCRIPTION
**Pull request by ShubhamJagtap2000**
## Related Issue

- Information about the related issue

Closes: #49, #52  

### Describe the changes you've made

I have changed the YouTube video resources you added in the [Resources](https://github.com/ShubhamJagtap2000/blockchain-dev-path/blob/frames/docs/%F0%9F%A4%A9%20Resources) file of this repo.

I have also added the video title (alt text) so that one can know the title of the video as well.

## Type of change

Improvement in Markdown, making content more interactive and exclusive. Styling and formatting update is done.
What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update.
- [x] This change requires a documentation update

## How Has This Been Tested?

Yes, I have edited some HTML code to show a thumbnail of the Youtube video link whereas there were only text links before.

Also, I have kept previous code as comment in case you want to edit it more.
I have tested the links to each video.
## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 Original           | Updated
 :--------------------: |:--------------------:
 
image
 | image |
 
## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/🤩 Resources/Bitcoin working.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
-Check out how bitcoin actually works.
-<iframe width="800" height="400" src="https://www.youtube.com/embed/bBC-nXj3Ng4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+**Check out how bitcoin actually works**
+
+Click below to see the video
+<!--iframe width="800" height="400" src="https://www.youtube.com/embed/bBC-nXj3Ng4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>-->
+
+[<img src="https://i.ytimg.com/vi/bBC-nXj3Ng4/maxresdefault.jpg" width="50%">](https://www.youtube.com/watch?v=bBC-nXj3Ng4 "How bitcoin works")
```
Comment (Panquesito7) [inline docs/🤩 Resources/Bitcoin working.md:6]:
```suggestion
[![Bit coin working](https://i.ytimg.com/vi/bBC-nXj3Ng4/maxresdefault.jpg)](https://www.youtube.com/watch?v=bBC-nXj3Ng4)
```

File: docs/🤩 Resources/Future of Web3.md [hunk 2]
```diff
@@ -1,2 +1,6 @@
-This content explains the INSANE future of Web 3.0 
-<iframe width="1263" height="480" src="https://www.youtube.com/embed/TV7SHUGTxNU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+**This content explains the INSANE future of Web 3.0**
+
+Click below to see the video
+<!--iframe width="1263" height="480" src="https://www.youtube.com/embed/TV7SHUGTxNU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>-->
+
+[!<img src="https://i.ytimg.com/vi/TV7SHUGTxNU/maxresdefault.jpg" width="50%">](https://www.youtube.com/watch?v=TV7SHUGTxNU "Future of Web 3.0")
```
Comment (Panquesito7) [inline docs/🤩 Resources/Future of Web3.md:6]:
```suggestion
[![Future of Web3](https://i.ytimg.com/vi/TV7SHUGTxNU/maxresdefault.jpg)](https://www.youtube.com/watch?v=TV7SHUGTxNU)
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
You might want to do the same with other instances as well.

**Comment by ShubhamJagtap2000** [comment]
> You might want to do the same with other instances as well.

I just committed the requested changes. Can we resolve this now or more changes are required?

File: docs/🤩 Resources/Guide to Full Stack Ethereum Development.md [hunk 3]
```diff
@@ -7,4 +7,6 @@ We’ll then connect the React front end to allow users to read data from the bl
 Finally, we’ll learn how to create and mint real ERC20 tokens, as well as how to rapidly prototype smart contracts using the Remix IDE.
 
 By the end of this guide, you should have a good understanding of how to build modern dApps with React and Solidity on the Ethereum blockchain.
-<iframe width="800" height="400" src="https://www.youtube.com/embed/a0osIaAOFSE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<!--iframe width="800" height="400" src="https://www.youtube.com/embed/a0osIaAOFSE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>-->
+
+[<img src="https://i.ytimg.com/vi/a0osIaAOFSE/maxresdefault.jpg" width="50%">](https://www.youtube.com/watch?v=a0osIaAOFSE "Ethereum development guide")
```
Comment (Panquesito7) [inline docs/🤩 Resources/Guide to Full Stack Ethereum Development.md:12]: Still not solved here.

File: docs/🤩 Resources/Roadmap to learn Blockchain development.md [hunk 4]
```diff
@@ -1,4 +1,14 @@
...
+
+<br>
+<br>
+<br>
+
 You may also try:-
-<iframe width="800" height="400" src="https://www.youtube.com/embed/P87pLayUD8c" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<!--iframe width="800" height="400" src="https://www.youtube.com/embed/P87pLayUD8c" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>-->
+
+[<img src="https://i.ytimg.com/vi/P87pLayUD8c/maxresdefault.jpg" width="50%">](https://www.youtube.com/watch?v=P87pLayUD8c "Blockchain roadmap")
```
Comment (Panquesito7) [inline docs/🤩 Resources/Roadmap to learn Blockchain development.md:14]: Same here.

File: docs/🤩 Resources/Stack NFT Marketplace on Ethereum with Polygon and Next.js.md [hunk 5]
```diff
@@ -4,4 +4,6 @@ In this video, you'll learn how to build a full stack NFT marketplace on Ethereu
 We'll start from scratch, creating a new project and installing the dependencies. We'll then write and test out the smart contracts. Once the tests have passed, we'll write the front end code to connect the smart contracts.
 
 After testing on a local network, we'll deploy to the Matic / Polygon network using a custom RPC provider (Infura).
-<iframe width="800" height="400" src="https://www.youtube.com/embed/GKJBEEXUha0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<!--iframe width="800" height="400" src="https://www.youtube.com/embed/GKJBEEXUha0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>-->
+
+[<img src="https://i.ytimg.com/vi/GKJBEEXUha0/maxresdefault.jpg" width="50%">](https://www.youtube.com/watch?v=GKJBEEXUha0 "Click here to see the video")
```
Comment (Panquesito7) [inline docs/🤩 Resources/Stack NFT Marketplace on Ethereum with Polygon and Next.js.md:9]: Same here.

File: docs/🤩 Resources/Web3 Stack.md [hunk 6]
```diff
@@ -1,2 +1,6 @@
  In this talk you'll learn what web3 is, and the tools and protocols needed to get started building modern, full stack decentralized applications.
-<iframe width="800" height="400" src="https://www.youtube.com/embed/f9XRH7bjV8M" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+ 
+ Click below to see the video
+ 
+[<img src="https://i.ytimg.com/vi/f9XRH7bjV8M/maxresdefault.jpg" width="50%">](https://www.youtube.com/watch?v=f9XRH7bjV8M "Web3 Stack")
```
Comment (Panquesito7) [inline docs/🤩 Resources/Web3 Stack.md:5]: And lastly, in here.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: docs/🤩 Resources/Roadmap to learn Blockchain development.md [hunk 7]
```diff
@@ -1,4 +1,14 @@
-Full Roadmap to learn Blockchain development 
+**Full Roadmap to learn Blockchain development**
+
+Click below to see the video
+
 <iframe width="800" height="400" src="https://www.youtube.com/embed/ci_AIMCF-HA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (Panquesito7) [inline docs/🤩 Resources/Roadmap to learn Blockchain development.md:5]: I think we should remove this. It doesn't even show up well on Markdown.

image

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: docs/🤩 Resources/Roadmap to learn Blockchain development.md (hunk 7 above)
Comment (ShubhamJagtap2000) [reply docs/🤩 Resources/Roadmap to learn Blockchain development.md:5]: Actually, that was already included in the README, and it's YouTube image has been corrupt so cannot display as we did for others. Will remove this as you say.

**Review by mkubdev — APPROVED** [review]
Hi, thanks for this changes! LGTM

File: docs/🤩 Resources/Roadmap to learn Blockchain development.md (hunk 7 above)
Comment (ShubhamJagtap2000) [reply docs/🤩 Resources/Roadmap to learn Blockchain development.md]: Commented it, tell me if I should delete all the commented content from all the files. Thank you.

**Comment by ShubhamJagtap2000** [comment]
> Hi, thanks for this changes! LGTM

Thank you.

**Review by shuklaritvik06 — APPROVED** [review]

**Comment by Panquesito7** [comment]
Seems the CI is failing due to various HTML codes that are invalid.
Should we remove them? We're not going to use them anyways.

**Comment by ShubhamJagtap2000** [comment]
> Seems the CI is failing due to various HTML codes in the Markdown files that are invalid. Should we remove them? We're not going to use them anyways.

You mean unwanted commented code?

**Comment by Panquesito7** [comment]
> > Seems the CI is failing due to various HTML codes in the Markdown files that are invalid. Should we remove them? We're not going to use them anyways.
> 
> You mean unwanted commented code?

Yep.

**Comment by ShubhamJagtap2000** [comment]
> > > Seems the CI is failing due to various HTML codes in the Markdown files that are invalid. Should we remove them? We're not going to use them anyways.
> > 
> > 
> > You mean unwanted commented code?
> 
> Yep.

Going to do it

**Comment by ShubhamJagtap2000** [comment]
> > > Seems the CI is failing due to various HTML codes in the Markdown files that are invalid. Should we remove them? We're not going to use them anyways.
> > 
> > 
> > You mean unwanted commented code?
> 
> Yep.

Just did the changes. Please review.

**Comment by ShubhamJagtap2000** [comment]
Thank you so much. Glad to co-author with you @shuklaritvik06 @Panquesito7 @mkubdev. Looking forward to future contributions.

**Comment by ShubhamJagtap2000** [comment]
Can I be a member of this org? Provided I haven't started in the blockchain field yet.

**Comment by shuklaritvik06** [comment]
> Can I be a member of this org? Provided I haven't started in the blockchain field yet.

Yes you can, in fact everyone can, there is nothing like I haven't started, you can come, learn and grow with us together, let's make WebXDAO much more grand ❤️

Just open an issue in start here repo with the template invite me to WebXDAO bot will send you an invite.

**Pull request merged**